.vitepress/config.mts

```diff
@@ -53,11 +53,16 @@ export default defineConfig({
     // https://vitepress.dev/reference/default-theme-config
     nav: [
       { text: 'Home', link: '/' },
-      { text: 'VectorChord Docs', link: '/vectorchord/getting-started/overview' },
-      { text: 'pgvecto.rs Docs', link: '/getting-started/overview' },
+      { text: 'Docs', link: '/vectorchord/' },
       { text: 'Cloud', link: '/cloud/' },
-      { text: 'Reference', link: '/reference/' },
       { text: 'Blog', link: 'https://blog.vectorchord.ai/' },
+      {
+        text: 'Others',
+        items: [
+          { text: 'pgvecto.rs Docs', link: '/getting-started/overview' },
+          { text: 'Reference', link: '/reference/' },
```

Comment:
It's docs for `pgvecto.rs Docs`, so it should not be a top-level directory

Reply:
The recommendation has been accepted and the revision has been completed.